# Module 2 Lab 1: Python Warm-Up + First Push

**Name:** Azra

This notebook works through Steps 1–6 of the Module 2 Lab 1 assignment: core Python constructs, a dictionary, and loading/inspecting `chickwts.csv` with pandas.

## Step 1: Wages with f-strings

In [1]:
name = "Azra"
hourly_wage = 32.00
hours = 20
weekly_pay = hourly_wage * hours
print(f"{name} earns {weekly_pay} per week")

Azra earns 640.0 per week


## Step 2: Count the warm days

In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]
warm_days = 0
for t in temps:
    if t > 65:
        warm_days = warm_days + 1
print(warm_days)

4


## Step 3: Write a temperature converter

In [3]:
def to_celsius(f):
    return (f - 32) * 5 / 9

print(round(to_celsius(temps[0]), 1))

16.1


## Step 4: A movie dictionary

In [4]:
movie = {"title": "Inception", "year": 2010, "rating": 8.8}

for key, value in movie.items():
    print(key, ":", value)

title : Inception
year : 2010
rating : 8.8


## Step 5: Load and inspect chickwts.csv

In [5]:
import pandas as pd

df = pd.read_csv("chickwts.csv")
df.head()

,weight,feed
0,179,horsebean
1,160,horsebean
2,136,horsebean
3,227,horsebean
4,217,horsebean


In [6]:
df.shape

(71, 2)

In [7]:
df.describe()

,weight
count,71.000000
mean,261.309859
std,78.073700
min,108.000000
25%,204.500000
50%,258.000000
75%,323.500000
max,423.000000


## Step 6: Interpret describe()

It tells me that the average weight of chicks was approximately 261 grams, and since the middle value is 258 g, it's close to average. Also it indicates that the distribution of weights is fairly wide and somewhat non-random. The lowest weight recorded was 108 grams while the highest was 423 grams. This suggests there could be significant differences in how well the various feeds work with regard to growth. To get a clearer picture of these differences, I would have to break down the data into each feed type rather than treating them as a whole.

## Appendix: AI Use Disclosure

**Tool used:** Claude (Anthropic), via Claude Code / Cowork.

**How it was used:** Used Claude to help build the code cells for Steps 1-5, following the patterns from the lab's tutorials. I reviewed and ran every cell myself and chose my own values for Step 1. I wrote the Step 6 interpretation myself.

**Key prompts:**
- "Build the Module 2 Lab 1 notebook (steps 1-5) following the tutorial patterns exactly, using chickwts.csv."